examples/checkout/nextjs-ui/app/globals.css

```
@@ -0,0 +1,787 @@
+:root {
+  --bg: #f5f7fb;
+  --surface: #ffffff;
```

Stylelint (17.12.0)

[error] Expected "`#ffffff`" to be "`#fff`" (color-hex-length)

(color-hex-length)